.ci/write-dockerfile.sh

```diff
@@ -347,7 +347,8 @@ RUN cd /new && rm -rf .git && \
         rm -rf /sage/src;                                    \
         mv src /sage/src;                                    \
         cd /sage && ./bootstrap && ./config.status;          \
-    fi
+    fi; \
+    cd /sage && rm -rf /new .git
 
```

Comment:
Is line 351 different from line 267 in what they do?

Reply:
Same thing, except line 351 starts out with /new being the current working directory, so I `cd` out of it first out of superstition

Reply:
I see. Reasonable (not just superstition).